packages/dds/tree/src/simple-tree/schemaFactory.ts

```diff
@@ -4,6 +4,10 @@
  */
 
 import { assert, unreachableCase } from "@fluidframework/core-utils/internal";
+// Include this unused import to avoid TypeScript generating an inline import for IFluidHandle in the d.ts file
+// which degrades the API-Extractor report quality since API-Extractor can not tell the inline import is the same as the non-inline one.
+// eslint-disable-next-line unused-imports/no-unused-imports
```

Comment:
Can we link to the issue here? I believe we have elsewhere we've done this

Reply:
Do you know what issue it is? I think this is technically a distinct problem from the one where different module entry points show up separate, but maybe not?